src/main/java/com/example/taskl12jpa/dao/entity/BooksEntity.java

```diff
@@ -0,0 +1,22 @@
+package com.example.taskl12jpa.dao.entity;
+
+import lombok.*;
```

Comment:
please don't use wildcard imports and change IDE settings: 
[ex](https://www.baeldung.com/intellij-disable-wildcard-import)